src/brewHandler.h

```diff
@@ -55,15 +55,14 @@ boolean brewPIDDisabled = false; // is PID disabled for delay after brew has sta
 
 // Shot timer with or without scale
 #if FEATURE_SCALE == 1
+float weight = 0;        // value from HX711
+float weightPreBrew = 0; // value of scale before wrew started
+float weightBrew = 0;    // weight value of brew
+bool scaleFailure = false;
 boolean scaleCalibrationOn = 0;
 boolean scaleTareOn = 0;
 int shottimerCounter = 10;
 float calibrationValue = SCALE_CALIBRATION_FACTOR; // use calibration example to get value
-float weight = 0;                                  // value from HX711
-float weightPreBrew = 0;                           // value of scale before wrew started
-float weightBrew = 0;                              // weight value of brew
-float scaleDelayValue = 2.5;                       // value in gramm that takes still flows onto the scale after brew is stopped
-bool scaleFailure = false;
```

Comment:
Why?

Reply:
this comes from moving these vars out of the FEATURE_SCALE flag and back into it, could be done cleaner -> not touching them at all in the end.
We changed our minds a lot douring this pr